
project system/core/
diff --git a/mkbootimg/mkbootimg.c b/mkbootimg/mkbootimg.c
index 3642647..5d92ca0 100644
--- a/mkbootimg/mkbootimg.c
+++ b/mkbootimg/mkbootimg.c
@@ -104,7 +104,7 @@ int main(int argc, char **argv)
     char *cmdline = "";
     char *bootimg = 0;
     char *board = "";
-    unsigned pagesize = 2048;
+    unsigned pagesize = 4096;
     int fd;
     SHA_CTX ctx;
     uint8_t* sha;
